vocabularies/Offline.xml

```diff
@@ -49,7 +49,21 @@ Please refer to the [Offline OData product home](https://help.sap.com/doc/f53c64
            attempt to upload any changes that are made to them locally.
            The service as implemented on the server is described by the CSDL document if
            all elements tagged as `Offline.ClientOnly` are removed together with all subsequently dangling references.</String>
-        </Annotation>        
+        </Annotation>
+      </Term>
+
+      <Term Name="LocalDraft" Type="Core.Tag" DefaultValue="true" Nullable="false" AppliesTo="EntitySet">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated model is draft and exists only on client devices. A draft entity
```

Comment:
No, LocalDraft does not imply ClientOnly.
LocalDraft does imply that there is a sever entity set(aka:non-draft) which reference to the same entity type as the draft.

Reply:
Your description says "The annotated model is draft and exists only on client devices".

This means that each draft model is client-only.

Please adapt the description if it is not correct.

Reply:
Here's a possible alternate Core.Description:

"Marks a local draft version of an entity set, which uses the same entity type as a non-draft entity set defined in the backend defined metadata."

Anyone needing more clarity can read the LongDescription.

Reply:
Thanks a lot,  the short description is updated to avoid ambiguity.

Reply:
> "Marks a local draft version of an entity set, which uses the same entity type as a non-draft entity set defined in the backend defined metadata."

If two backend-defined entity sets use the same entity type and a third entity set tagged as `Offline.LocalDraft` uses the same entity type, it is unclear to which of the two other entity sets it corresponds. Should `Offline.LocalDraft` perhaps contain the name of the backend-defined entity set rather than being a boolean?

Reply:
Sorry, I still don't understand. You say that a `LocalDraft` entity set need not be `ClientOnly`. But then it exists also on the server, right? Can you give an example for this?

Reply:
Hello Heiko,

Sorry for the confusion,  LocalDraft and ClientOnly are different concept here.
OfflineOData is basically client-server structure. Users download server data to client device, and then make all odata CRUD request against local dataset. (This is a big difference with online-odata which operates on back-end dataset directly).

The odata metadata for OfflineOData client device is a combination of metadata downloaded from back-end and metadata defined from the client side. ClientOnly and LocalDraft are two terms defined in the client side. ClientOnly means some data which exists in client device only, and should never upload to back-end. The entity type annotated with ClientOnly could be defined from the client side,  which back-end may not aware of. 
The LocalDraft indicates that the underlying entity type is recognizable by back-end, and the draft could be covered to non-draft and upload to back-end later.

Reply:
For me this means that
* The local dataset on the client can contain both draft and non-draft entities.
* Draft entities can exist in the local dataset on the client as well as on the back-end.

Correct?

Reply:
That's correct, thanks. Just to mention that the draft entities are converted to non-draft entities from the client side  before upload to back-end, and the conversion is done by client app which is transparent to back-end. So technically back-end will only see non-draft entities, thanks.

Reply:
> So technically back-end will only see non-draft entities, thanks.

But why is the LocalDraft entity set (`SalesOrderDraft` in your example) then needed in the back-end?

Reply:
Hi Heiko,

I though the sample is used to demonstrate the usage the term.
If I'm wrong I will delete all the samples, sorry for my misunderstanding.

Reply:
My question is not only about your sample: If the back-end will only see non-draft entities, why is the LocalDraft entity set then needed in the back-end?

Reply:
I don't think LocalDraft entity set is needed in the back-end. It is the client app which needs the LocalDraft entity set, thanks.

Reply:
> why is the LocalDraft entity set then needed in the back-end?

My understanding: the LocalDraft entity set exists only locally on the client, and the LocalDraft-sample shows the client-local schema.

@bruce-qin-dev @HeikoTheissen Should we have both `LocalDraft-sample-client` and `LocalDraft-sample-server` files to show the difference?